docs/lib/auth/fragments/js/social.md

```diff
@@ -527,3 +536,35 @@ Amplify.configure({
     // ...
 });
 ```
+
+### React Native Specifics
```

Comment:
This pattern should only really apply for Expo, right? Regular React Native apps won't need to change their schemes. I'd suggest changing the title of this section to just be regarding Expo instead of React Native as a whole.